src/coreclr/nativeaot/Runtime.Base/src/System/Runtime/ExceptionHandling.wasm.cs

```diff
@@ -92,7 +92,7 @@ public bool ContainsCodeOffset(uint idxTryLandingStart)
                 }
                 else
                 {
-                    tryRegionIdx = 0;
+                    tryRegionIdx = MaxTryRegionIdx;
```

Comment:
Is this needed? We have set `tryRegionIdx = MaxTryRegionIdx;` before the loop.

Reply:
No, deleted.  Thanks